### ISSUE TITLE
Wrong offset: 1926. Should be in range: [0, 1921]

### ISSUE THREAD

**Issue by moniuch** [denofevil/import-cost#5]
Wrong offset: 1926. Should be in range: [0, 1921]

Before that happened I turned off Import Cost and postponed WS reboot.
(I turned Import Cost, because the numbers it places are red, and unneccessarily putting my eyes to alert. Did not find any styling options to make them more subtle)

**Comment by igal1c0de4n** [comment]
Probably similar issue on my end:
```
java.lang.IndexOutOfBoundsException: Wrong offset: 275. Should be in range: [0, 249]
	at com.intellij.openapi.editor.impl.LineSet.findLineIndex(LineSet.java:172)
	at com.intellij.openapi.editor.impl.DocumentImpl.getLineNumber(DocumentImpl.java:1008)
	at com.github.denofevil.importCost.LanguageService$processImports$1$1.visitES6ImportDeclaration(LanguageService.kt:133)
	at com.intellij.lang.ecmascript6.psi.impl.ES6ImportDeclarationImpl.accept(ES6ImportDeclarationImpl.java:40)
	at com.intellij.psi.PsiWalkingState.visit(PsiWalkingState.java:70)
	at com.intellij.psi.PsiWalkingState.visit(PsiWalkingState.java:27)
	at com.intellij.util.WalkingState.walkChildren(WalkingState.java:65)
	at com.intellij.util.WalkingState.elementStarted(WalkingState.java:52)
	at com.intellij.psi.PsiWalkingState.elementStarted(PsiWalkingState.java:79)
	at com.intellij.lang.javascript.psi.JSRecursiveWalkingElementVisitor.visitElement(JSRecursiveWalkingElementVisitor.java:29)
	at com.intellij.psi.PsiElementVisitor.visitFile(PsiElementVisitor.java:34)
	at com.intellij.lang.javascript.psi.JSElementVisitor.visitJSFile(JSElementVisitor.java:309)
	at com.intellij.lang.javascript.psi.impl.JSFileImpl.accept(JSFileImpl.java:112)
	at com.github.denofevil.importCost.LanguageService$processImports$1.run(LanguageService.kt:119)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:948)
	at com.github.denofevil.importCost.LanguageService.processImports(LanguageService.kt:117)
	at com.github.denofevil.importCost.LanguageService.access$processImports(LanguageService.kt:34)
	at com.github.denofevil.importCost.LanguageService$updateImports$1.run(LanguageService.kt:107)
	at com.intellij.util.ui.update.MergingUpdateQueue.execute(MergingUpdateQueue.java:336)
	at com.intellij.util.ui.update.MergingUpdateQueue.execute(MergingUpdateQueue.java:326)
	at com.intellij.util.ui.update.MergingUpdateQueue.lambda$flush$1(MergingUpdateQueue.java:276)
	at com.intellij.util.ui.update.MergingUpdateQueue.flush(MergingUpdateQueue.java:290)
	at com.intellij.util.ui.update.MergingUpdateQueue.run(MergingUpdateQueue.java:245)
	at com.intellij.util.concurrency.QueueProcessor.runSafely(QueueProcessor.java:232)
	at com.intellij.util.Alarm$Request.runSafely(Alarm.java:356)
	at com.intellij.util.Alarm$Request.run(Alarm.java:343)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at com.intellij.util.concurrency.SchedulingWrapper$MyScheduledFutureTask.run(SchedulingWrapper.java:228)
	at com.intellij.util.concurrency.BoundedTaskExecutor.doRun(BoundedTaskExecutor.java:227)
	at com.intellij.util.concurrency.BoundedTaskExecutor.access$100(BoundedTaskExecutor.java:26)
	at com.intellij.util.concurrency.BoundedTaskExecutor$2$1.run(BoundedTaskExecutor.java:200)
	at com.intellij.util.ConcurrencyUtil.runUnderThreadName(ConcurrencyUtil.java:229)
	at com.intellij.util.concurrency.BoundedTaskExecutor$2.run(BoundedTaskExecutor.java:194)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
```

**Comment by anstarovoyt** [comment]
fixed in 1.1.2023.
